crates/crates_io_database_dump/src/dump-db.toml

```diff
@@ -48,6 +48,9 @@ description = "public"
 crates_cnt = "public"
 created_at = "public"
 path = "public"
+[categories.sequence]
+column = "id"
+name = "categories_id_seq"
```

Comment:
instead of manually declaring them here, would it be possible to derive them from the database schema in some way?

Reply:
This is doable as follows, which is modified from https://stackoverflow.com/a/55414721:
```sql
select tbl.relname as table_name, 
       col.attname as column_name,
       s.relname as sequence_name
from pg_class s
  join pg_namespace sn on sn.oid = s.relnamespace 
  join pg_depend d on d.refobjid = s.oid and d.refclassid='pg_class'::regclass 
  join pg_attrdef ad on ad.oid = d.objid and d.classid = 'pg_attrdef'::regclass
  join pg_attribute col on col.attrelid = ad.adrelid and col.attnum = ad.adnum
  join pg_class tbl on tbl.oid = ad.adrelid 
  join pg_namespace ts on ts.oid = tbl.relnamespace 
where s.relkind = 'S'
--  and s.relname = 'your_sequence_name_her'
  and d.deptype in ('a', 'n');
```


From the result, you should see something similar to the following:
```
      table_name       | column_name |        sequence_name         
-----------------------+-------------+------------------------------
 api_tokens            | id          | api_tokens_id_seq
 background_jobs       | id          | background_jobs_id_seq
 categories            | id          | categories_id_seq
 crates                | id          | packages_id_seq
 deleted_crates        | id          | deleted_crates_id_seq
 dependencies          | id          | dependencies_id_seq
 emails                | id          | emails_id_seq
 keywords              | id          | keywords_id_seq
 teams                 | id          | teams_id_seq
 users                 | id          | users_id_seq
 version_owner_actions | id          | version_owner_actions_id_seq
 versions              | id          | versions_id_seq
(12 rows)
```